server/dist.js

```diff
@@ -15,7 +15,7 @@ const PORT = 3000;
 // pre-render
 const index = pug.renderFile('./src/index.pug', {
   dist: true,
-  // config: config.windowEnv()
+  config: config.windowEnv()
```

Comment:
You also need to add this to the dev server script.  I believe that `res.render('./src/index')` in there is calling to the pug renderer under the hood.  So you just need to pass a 2nd argument with `{config: config.windowEnv()}` to it, like they do in the [express docs](https://expressjs.com/en/guide/using-template-engines.html).